src/semidiscretization/semidiscretization_hyperbolic_parabolic.jl

```diff
@@ -0,0 +1,184 @@
+  mesh, equations, solver, cache = mesh_equations_solver_cache(semi.semi_hyperbolic)
+  @unpack initial_condition = semi.semi_hyperbolic
+
+  calc_error_norms(func, u, t, analyzer, mesh, equations, initial_condition, solver, cache, cache_analysis)
+end
+
+
+function compute_coefficients(t, semi::SemidiscretizationHyperbolicParabolic)
+  compute_coefficients(semi.semi_hyperbolic.initial_condition, t, semi.semi_hyperbolic)
+end
+
+function compute_coefficients!(u_ode::AbstractVector, t, semi::SemidiscretizationHyperbolicParabolic)
+  compute_coefficients!(u_ode, semi.semi_hyperbolic.initial_condition, t, semi.semi_hyperbolic)
+end
+
+
+function rhs!(du_ode, u_ode, semi::SemidiscretizationHyperbolicParabolic, t)
+  @unpack du_ode_parabolic = semi.cache
+
+  # TODO: Taal decide, do we need to pass the mesh?
+  time_start = time_ns()
+
+  # Resize the storage for the parabolic time derivative in case it was changed
+  resize!(du_ode_parabolic, length(du_ode))
+
+  @timeit_debug timer() "rhs!" begin
+    # Compute hyperbolic time derivative
+    rhs!(du_ode, u_ode, semi.semi_hyperbolic, t)
+
+    # Compute parabolic time derivative
+    rhs!(du_ode_parabolic, u_ode, semi.semi_parabolic, t)
+
+    # Add parabolic update to hyperbolic time derivative
+    du_ode .+= du_ode_parabolic
+  end
+
+  runtime = time_ns() - time_start
+  put!(semi.performance_counter, runtime)
+
+  return nothing
+end
+
+
+"""
+    SemidiscretizationHyperbolicParabolicBR1(mesh, equations, initial_condition, solver::DGSEM;
+                                             source_terms=nothing,
+                                             boundary_conditions=boundary_condition_periodic)
+
+Convenience function to construct a semidiscretization of a hyperbolic-parabolic PDE with the BR1
+scheme for a DGSEM solver. The passed arguments must correspond to the hyperbolic setup, including
+the `solver`. Internally, a matching solver for the parabolic system is created and both a
+hyperbolic and a parabolic semidiscretization are created and passed to a
+`SemidiscretizationHyperbolicParabolic`.
+"""
+function SemidiscretizationHyperbolicParabolicBR1(mesh, equations, initial_condition, solver::DGSEM;
+                                                  source_terms=nothing,
+                                                  boundary_conditions=boundary_condition_periodic,
+                                                  RealT=real(solver),
+                                                  volume_integral_parabolic=VolumeIntegralWeakForm())
+  semi_hyperbolic = SemidiscretizationHyperbolic(mesh, equations, initial_condition, solver;
+                                                 source_terms=source_terms,
+                                                 boundary_conditions=boundary_conditions,
+                                                 RealT=RealT)
+
+  solver_parabolic = DGSEM(solver.basis, flux_central, volume_integral_parabolic, solver.mortar)
```

Comment:
```suggestion
  solver_parabolic = DGSEM(solver.basis, flux_central, volume_integral_parabolic)
```
We should probably not use Jesse's EC/ES mortars for parabolic terms...